README.md

```diff
@@ -300,6 +300,9 @@ CLIENT_MAX_BODY_SIZE=1M                 # 0 disables checking request body size
 PROXY_BUFFERS="8 4k"                    # Either 4k or 8k depending on the platform
 PROXY_BUFFER_SIZE="4k"                  # Either 4k or 8k depending on the platform
 RESOLVER="Your custom solver string"
+PROXY_CONNECT_TIMEOUT=90;
```

Comment:
Default is actually 60 according to [Nginx documentation](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_connect_timeout).